packages/isUpperCase️/README.md

```diff
@@ -0,0 +1,22 @@
+## `isUpperCase`
+
+Checks whether the input is a string containing only uppercase characters
+
+`npm i @plexis/is-uppercase`
+
+**Usage**
+
+```javascript
+import isUpperCase from '@plexis/is-uppercase';
+
+isUpperCase️('b') // returns false
+isUpperCase️('B') // returns true
+isUpperCase️('BCD') // returns true
+```
+
+**Aliases**
+
+```javascript
+import {isUpperCase} from 'plexis';
```

Comment:
Can you please merge these two into one line?

Reply:
Sorry, I didn't understood